src/v3/src/hooks/usePolling.ts

```diff
@@ -100,6 +102,20 @@ export const usePolling = (
         stateHandle: stateToken && idxTransaction?.context?.stateHandle,
         ...payload,
       });
+
+      // @ts-expect-error OKTA-585869 errorCode & errorIntent properties missing from context type
+      if ((newTransaction?.context?.errorCode === 'E0000047' && !newTransaction?.context?.errorIntent)
```

Comment:
Can we also add a comment here about what this errorCode means and also what the check for errorIntent means?

Reply:
The errorIntent check was carried over from v2.  Not entirely sure why they check to make sure the errorIntent isnt there.  Ill see if I can find someone who knows why.